mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -245,6 +245,7 @@ LogicalResult ConvConverter<ConvType>::matchAndRewrite(
   auto outputTy = cast<MIXRShapedType>(results[0].getType());
   Type outElementTy = outputTy.getElementType();
   Type newOutElementTy = getTypeConverter()->convertType(outElementTy);
+  bool isBwdConvOp = isa<migraphx::ConvolutionBwdDataOp>(op);
```

Comment:
nit: because it can also be bwd_weights, 
```suggestion
  bool isBwdDataConvOp = isa<migraphx::ConvolutionBwdDataOp>(op);
```